include/pagmo/rng.hpp

```diff
@@ -57,7 +57,11 @@ using random_engine_type = std::mt19937;
  */
 struct PAGMO_DLL_PUBLIC random_device {
     static unsigned next();
-    static void set_seed(unsigned);
+
+#if !defined(PAGMO_DOXYGEN_INVOKED)
```

Comment:
Was this the cause of doxygen troubles??

Reply:
No idea, this is just the workaround for the current doxygen version (1.8.17). Other doxygen versions give errors in other places.